web/assets/js/talk/listeners.js

```diff
@@ -0,0 +1,44 @@
+//Favorite Listener
+$('.js-talk-favorite').on('click', function(e) {
+  var talk = new Talk($(this).data('id'), $(this));
+  e.preventDefault();
+  talk.favorite();
+});
+
+//Select Listener
+$('.js-talk-select').on('click', function(e) {
+  var talk = new Talk($(this).data('id'), $(this));
+  e.preventDefault();
+  talk.select();
+});
+
+//Rating Listeners
+//Yes No Rating listener
+$('.js-talk-rating-yes-no').on('click', function(e) {
+  var id = $(this).data('id');
+  var talk = new Talk($(this).data('id'), $(this), $(this).data());
+  var rating = $(this).data('rating');
+  e.preventDefault();
```

Comment:
What do you think about being a bit more generous with vertical whitespace here and elsewhere, for example:

```diff
diff --git a/web/assets/js/talk/listeners.js b/web/assets/js/talk/listeners.js
index 89343ccf..49ab2912 100644
--- a/web/assets/js/talk/listeners.js
+++ b/web/assets/js/talk/listeners.js
@@ -1,14 +1,18 @@
 //Favorite Listener
 $('.js-talk-favorite').on('click', function(e) {
   var talk = new Talk($(this).data('id'), $(this));
+
   e.preventDefault();
+
   talk.favorite();
 });

 //Select Listener
 $('.js-talk-select').on('click', function(e) {
   var talk = new Talk($(this).data('id'), $(this));
+
   e.preventDefault();
+
   talk.select();
 });

@@ -18,7 +22,9 @@ $('.js-talk-rating-yes-no').on('click', function(e) {
   var id = $(this).data('id');
   var talk = new Talk($(this).data('id'), $(this), $(this).data());
   var rating = $(this).data('rating');
+
   e.preventDefault();
+
   if (talk.rate(rating)) {
     if (rating === -1) {
       $('#talk-downvote-' + id + ' i').addClass('text-red-dark');
@@ -40,5 +46,6 @@ $('.js-talk-rating-yes-no').on('click', function(e) {
 //One to Ten Rating Listener.
 $('.js-talk-rating-one-to-ten').on('change', function () {
   var talk = new Talk($(this).data('id'), $(this));
+
   talk.rate($(this).val())
 });
```